qcodes/instrument/channel.py

```diff
@@ -1,10 +1,12 @@
 """ Base class for the channel of an instrument """
+from typing import List, Tuple, Union
```

Comment:
Not in general we have taken a soft approach and do it gradually where it makes sense. In this case my linter complained because it got confused about whether `_channels` is a tuple or a list so I tried to help it a bit. Don't mind removing it